source/making_payments/index.html.md.erb

```diff
@@ -286,15 +286,19 @@ Your user can select a link on the payment page to cancel a payment that’s in
 progress. Alternatively, a service can make the following API call for a given
 `paymentId`:
 
-`POST /v1/payments/paymentId/cancel`
+`POST /v1/payments/{PAYMENT_ID}/cancel`
+
+Replace `{PAYMENT_ID}` with the ID of the payment that you are cancelling.
 
 A `204` response indicates success. Any other response indicates an error.
 
-### Find out if you can cancel a payment
+### Check if you can cancel a payment
+
+Use a GET request against a single payment to check if you can cancel it:
```

Comment:
General point - minimise "it", "that" and "they" to minimise ambiguity. I would rephrase to "...cancel that payment." This is not a huge issue given that there are only 2 entities in the sentence, the GET request and the single payment, but it's a good habit to get into.

Also, should the GET be in `` i.e. `GET`?